packages/vscode-extension/src/service/ListFilesService.ts

```diff
@@ -130,38 +104,52 @@ export default class ListFilesService {
 		return;
 	}
 
-	public async listFiles(selectedFolder: string) {
-		const listfilesOptions: { [key: string]: string } = {};
-		listfilesOptions[LIST_FILES_COMMAND.OPTIONS.FOLDER] = selectedFolder;
+	public async getFilesFromSelectedAccountFileCabinetFolder(selectedFolder: string) {
+		const defaultAuthId = this.getDefaultAuthId();
+		if (!defaultAuthId) {
+			return;
+		}
 
-		const commandActionPromise = this.listFilesCommand(listfilesOptions);
-		const commandMessage = this.translationService.getMessage(COMMAND.TRIGGERED, commandsInfoMap.listfiles.vscodeCommandName);
+		const accountFileCabinetService = new AccountFileCabinetService(getSdkPath(), this.executionEnvironmentContext, defaultAuthId);
+		const listFilesPromise = accountFileCabinetService.listFiles(selectedFolder);
 		const statusBarMessage = this.translationService.getMessage(LIST_FILES.LISTING);
-		this.messageService.showInformationMessage(commandMessage, statusBarMessage, commandActionPromise);
+		this.messageService.showStatusBarMessage(statusBarMessage, listFilesPromise);
 
-		const actionResult = await commandActionPromise;
+		const actionResult = await listFilesPromise;
 		if (actionResult.status === actionResultStatus.SUCCESS) {
 			return actionResult.data;
 		} else {
 			throw actionResult.errorMessages;
 		}
 	}
 
-	private async listFilesCommand(listFilesOption: { [key: string]: string }) {
-		if (!this.vsConsoleLogger) {
-			throw Error(CONSOLE_LOGGER_ERROR);
+	private getDefaultAuthId(): string | undefined {
```

Comment:
Assuming we will have this logic in different places, (when default id does not exist) can we move this code to a global place where others can use?

Reply:
I was initially thinking about it, and I forgot about it later. I'll move it to another place where it will could be easy reused.

Reply:
Thanks David.